### PR TITLE
AC: update adapter configuration

### PR DESCRIPTION
**Pull request by eaidova**
remove users confusing behaviour

### CODE REVIEW

File: tools/accuracy_checker/accuracy_checker/adapters/yolo.py [hunk 1]
```diff
@@ -283,7 +283,7 @@ def parameters(cls):
             'threshold': NumberField(value_type=float, optional=True, min_value=0, default=0.001,
                                      description="Minimal objectiveness score value for valid detections."),
             'outputs': ListField(
-                optional=True, default=[],
+                optional=False,
```
Comment (eizamaliev) [inline tools/accuracy_checker/accuracy_checker/adapters/yolo.py:286]: By default `optional=False`, I think you should remove it.

File: tools/accuracy_checker/accuracy_checker/adapters/yolo.py [hunk 2]
```diff
@@ -283,7 +283,7 @@ def parameters(cls):
...
                 description="The list of output layers names (optional),"
                             " if specified there should be exactly 3 output layers provided."
```
Comment (eizamaliev) [inline tools/accuracy_checker/accuracy_checker/adapters/yolo.py:288]:
```suggestion
                description="The list of output layers names."
```

**Comment by eizamaliev** [comment]
What if user describes only two layer for yolo-v3? Should we test this case?

File: tools/accuracy_checker/accuracy_checker/adapters/yolo.py (hunk 1 above)
Comment (eaidova) [reply tools/accuracy_checker/accuracy_checker/adapters/yolo.py]: fixed

**Comment by eaidova** [comment]
> What if user describes only two layer for yolo-v3? Should we test this case?

It already tested by one user and it works :)

**Pull request merged**